noodles-gff/src/writer.rs

```diff
@@ -1,5 +1,7 @@
 use std::io::{self, Write};
 
+use crate::Line;
+
 use super::{Directive, Record};
```

Comment:
```suggestion
use super::{Directive, Line, Record};
```

Reply:
Good catch, this is an editor setting for me. My fault